components/ActionBarFeature/ActionBarFeature.js

```diff
@@ -41,18 +39,24 @@ const ActionBarFeature = props => {
               { action, icon, title, theme, relatedNode, ...actionProps },
               i
             ) => (
-              // <ThemeProvider key={i} themeMixin={theme}>
-              <CustomLink
-                as="a"
-                href={getUrlFromRelatedNode(relatedNode)}
-                Component={ActionBarItem}
-                {...(!icon ? {} : { icon })}
-                label={title}
-                onPressActionItem={e =>
-                  onPressActionItem(e, { action, relatedNode })
-                }
-              />
-              // </ThemeProvider>
+              /**
+               * ! Our ThemeProvider component from /ui-kit is not working properly so we are using the orignal component from the 'style-components' package.
+               */
+              <SCThemeProvider key={i} theme={theme}>
```

Comment:
Hey @jordanwade or @drewbarontini , any off-hand ideas why _our_ app's `ThemeProvider` wouldn't work here versus the underlying `SCThemeProvider`?

I'm fuzzy on the recent work around theming but the general pattern seems to be overriding the theme in different branches of the component tree.